config/routes.rb

```diff
@@ -1,12 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :posts
-  resources :likes
-  resources :comments
-  resources :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'users' => 'users#index'
+  get 'users/:id' => 'users#show'
+  get 'posts' => 'posts#index'
+  get 'posts/:id' => 'posts#show'
```

Comment:
- [ ] I think we can improve the definition of these routes. The post routes should be nested within the user routes so that we can easily access the user id whoever is trying to create a post.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```